app/src/layouts/DashboardLayout/MenuHeader/index.js

```diff
@@ -100,25 +104,28 @@ const MenuHeader = () => {
           <Col className="ml-auto">
             <div className="header-right-section">
               <Row align="middle" gutter={8} wrap={false}>
-                {appMode !== GLOBAL_CONSTANTS.APP_MODES.DESKTOP &&
-                user?.details?.planDetails?.status === "canceled" &&
-                isPlanExpiredBannerClosed ? (
-                  <div className="hidden-on-small-screen">
-                    <PlanExpiredBadge />
-                  </div>
-                ) : null}
-                <RQButton
-                  type="default"
-                  className="header-search-btn"
-                  onClick={() => dispatch(actions.updateIsCommandBarOpen(true))}
-                >
-                  <div>
-                    <SearchOutlined /> Search
-                  </div>
-                  <div className="search-shortcut-annotation">⌘+K</div>
-                </RQButton>
-                {/* TEMPORARILY HIDDEN  */}
-                {/* <Col className="hidden-on-small-screen">
+                <>
+                  {isAppTypeRequestly && (
+                    <>
+                      {appMode !== GLOBAL_CONSTANTS.APP_MODES.DESKTOP &&
```

Comment:
this check would become cleaner if constant is moved to [`common/constants.js` under `CONSTANTS.APP_MODES`](https://github.com/requestly/requestly/blob/6db0cbf3eb80ea230a530912e54d514122e44b5d/common/constants.js#L10-L16)